c2cgeoportal/scaffolds/update/CONST_alembic/main/versions/116b9b79fc4d_internal_and_external_layer_tables_.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2015, Camptocamp SA
+# All rights reserved.
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+"""internal and external layer tables refactoring, new ogc table
+
+Revision ID: 116b9b79fc4d
+Revises: 1418cb05921b
+Create Date: 2015-10-28 12:21:59.162238
+"""
+
+from alembic import op, context
+from sqlalchemy import ForeignKey, Column, Table, MetaData
+from sqlalchemy.types import Integer, Boolean, Unicode, String, Float, \
+    UserDefinedType, DateTime, Enum
+
+# revision identifiers, used by Alembic.
+revision = '116b9b79fc4d'
+down_revision = 'a4f1aac9bda'
+branch_labels = None
```

Comment:
ERROR [alembic.util] Branch name 'master' in revision a4f1aac9bda already used by revision 116b9b79fc4d
  FAILED: Branch name 'master' in revision a4f1aac9bda already used by revision 116b9b79fc4d